setup.py

```diff
@@ -1,24 +1,7 @@
-import os
-import sys
-
 from setuptools import find_packages
 from setuptools import setup
 
 
-# setup logic from github.com/jupyterhub/jupyterhub
-v = sys.version_info
-if v[:2] < (3, 6):
-    error = "ERROR: LTIAuthenticator requires Python version 3.6 or above."
-    print(error, file=sys.stderr)
-    sys.exit(1)
```

Comment:
Hmmm removing this section may have been irrelevant. I noted `python_requires=">3.6"` in the setup() call below and thought it was something related to versions and irrelevant.